san/transform.py

```diff
@@ -32,7 +37,18 @@ def transform_query_result(idx, query, data):
 def eth_top_transactions_transform(data):
     for column in data:
         column['fromAddressIsExchange'] = column['fromAddress']['isExchange']
-        column['toAddressIsExchange'] = column['toAddress']['isExchange']
         column['fromAddress'] = column['fromAddress']['address']
+        column['toAddressIsExchange'] = column['toAddress']['isExchange']
         column['toAddress'] = column['toAddress']['address']
+        column['toAddress'] = column.pop('toAddress')
+        column['toAddressIsExchange'] = column.pop('toAddressIsExchange')
+        column['trxHash'] = column.pop('trxHash')
+        column['trxValue'] = column.pop('trxValue')
```

Comment:
there is no return here ?

Reply:
You're right, I will add it now, though it is kind of a void function

Reply:
These also